src/test/scala/cognite/spark/v1/AssetsRelationTest.scala

```diff
@@ -145,9 +146,13 @@ class AssetsRelationTest extends FlatSpec with Matchers with SparkTest {
     val assetsCreated = getNumberOfRowsCreated(metricsPrefix, "assets")
     assert(assetsCreated == 1)
 
-    retryWhile[Array[Row]](
+    val Array(createdAsset) = retryWhile[Array[Row]](
       spark.sql(s"select * from assets where source = '$assetsTestSource'").collect,
       rows => rows.length < 1)
+
+    assert(createdAsset.getAs[String]("name") == "asset name")
+    assert(createdAsset.getAs[Long]("dataSetId") == testDataSetId)
+    assert(createdAsset.getAs[String]("externalId") == "1")
```

Comment:
We should use the `x should be(y)` or `x shouldBe y` matchers when asserting equality.

```suggestion
    createdAsset.getAs[String]("name") shouldBe "asset name"
    createdAsset.getAs[Long]("dataSetId") shouldBe testDataSetId
    createdAsset.getAs[String]("externalId") shouldBe "1"
```

Reply:
We should also perform the same checks for other resource types (events, time series, etc.).

Reply:
If you mean... However, I managed to find a disadvantage of `shouldBe` - when I get the types wrong (for example `.getAs[Option[Long]]("x") == 1`), Scala compiler complain. It just compiles and fails the test when I use use `shouldBe` :/

Reply:
Agh, that's a good point. Feel free to disregard this comment then.

Reply:
I would still like to see tests for other resource types though :)